src/sql-editor/standardSql/types.ts

```diff
@@ -11,7 +18,7 @@ export interface SuggestionsRegistryItem extends RegistryItem {
 export interface MacrosRegistryItem extends RegistryItem {
   type: MacroType;
   text: string;
-  args?: string[];
+  args: string[];
```

Comment:
can you talk through what these args would be? might be nice to give examples for all of these with helpful comments

Reply:
This is not really part of this PR - I just changed this field here so that it matches the typings [here](https://github.com/grafana/grafana-experimental/blob/standard-sql-completion-item-provider/src/sql-editor/types.ts#L82). But I believe a macro function can have a set of args, and if you specify them here they will be suggested when inside the macro func.